src/index.tsx

```diff
@@ -327,30 +420,62 @@ const styles = StyleSheet.create({
   flexOne: {
     flex: 1,
   },
-  flatList: {
+  flatListH: {
     flex: 1,
     flexDirection: isAndroidRTL ? 'row-reverse' : 'row',
   },
-  paginationContainer: {
+  flatListV: {
+    flex: 1,
+  },
+  paginationContainerH: {
     position: 'absolute',
     bottom: 16,
     left: 16,
     right: 16,
     justifyContent: 'center',
   },
-  paginationDots: {
+  paginationContainerV: {
+    position: 'absolute',
+    bottom: 16,
+    right: 8,
+    top: 16,
+    justifyContent: 'center',
+  },
```

Comment:
Looks like there's no difference between these two, so I'd prefer to just have a single `paginationContainer`

Reply:
there is difference horizontal pagination has bottom, left, right configurations while vertical pagination has other configuration